src/types/fsxa-ui.d.ts

```diff
@@ -1,4 +1,5 @@
 export * from "./internal";
 export * from "./components";
 import Sections from "./sections";
+export * from "./sections";
```

Comment:
What is this export good for? If needed at all, can't it replace the following specific export of "Sections"?

Reply:
At the moment you can import the section components with JSX only if you use the following expression for it: 
`import { Section.AccordioinSection } from "fsxa-ui`

For classic Vue Projects it is even worse: 
`import { Section } from "fsxa-ui` and
`const { AccordionSection } = Section`

Because there is no benefit in doing this with JSX and only causing trouble with classic vue we decided to export the section components like the other components. 

However, as a transitional phase, both export options will be retained and at a later date the older variant will be removed.